ci/scripts/utils/launch_java_agent.sh

```diff
@@ -142,6 +143,9 @@ print(data.get('offline','True'))
 EOF
 chmod u+x parse.py
 
+# Needed for Gaea as it has C capatalized on the Node names in Jenkins
+MACHINE_ID="${MACHINE_ID//c/C}"
```

Comment:
Can we ask Kris to not do that.  i.e. ask him to name the node `Gaeac6-EMC` instead of `GaeaC6-EMC`?

Reply:
Ok, I updated this to `MACHINE_ID="$(echo "$MACHINE_ID" | sed 's/c\([0-9]\)/C\1/g')"` to account for Hercules, B=but I will still follow up with Kris so we can remove it.

Reply:
Done, Kris made the update and we restarted the connection without the kludge.